### ISSUE TITLE
Terokkar spirit towers don't reset to neutral

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#5275]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When the timer on the spirit towers in Terokkar Forest expires, the towers stay capped to whatever faction had them before. 

**Expected behaviour**: Tell us what should happen instead
When the timer runs out, the spirit towers are suppose to reset to neutral. 
**Steps to reproduce the problem**:

1. Cap all towers
2. Wait out timer.. 
3. Watch towers not resetting to neutral

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
I missed to take screenshots the last time I witnessed this, but I will take some next time and update this ticket. 
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexandru58** [comment]
Confirmed, they are not reseting to Neutral... Developers, please, fix this!
